prime-angular-frontend/src/app/modules/adjudication/shared/components/health-authority-table/health-authority-table.component.scss

```diff
@@ -7,3 +7,7 @@
 .red-icon {
   color: theme-palette(red);
 }
+
```

Comment:
Also I know that the hamburger menus were yellow in the mockup, but would it make more sense to have them look the same as the rest of pages?

Reply:
I think I'd leave it to the end users to decide as it was quite explicit in the Jira ticket:  "the hamburger menu on the organization level is another color (yellow for now)"

Reply:
Realized light blue background was in mockup after I looked at mockup on my main monitor rather than my smaller (and older) right-side monitor!